### DISCUSSION TITLE
We still can't follow organizations on the mobile app

### DISCUSSION THREAD

**Discussion by EDM115** [community/community#15077] [Mobile] [answered]
![](https://telegra.ph/file/1b507bc6ef413cb2c82d5.png)  
  
As shown here, org’s appear, but not the Follow button 😭  
  
*Note : I'm on `v.1.61.0-beta`*

**Comment by stevepopovich** [comment, marked as answer]
Hi @EDM115, you are not the only one looking for this! Adding this to our tally of requests and passing to the team for prioritization, thanks!

**Reply by EDM115** [reply]
Have been added on `v1.73.0-beta`

**Reply by stevepopovich** [reply]
Ah yep, glad you saw.
